src/cli.rs

```diff
@@ -88,33 +132,40 @@ pub struct WatchOpts {
     )]
     pub debounce: Duration,
 
-    /// A path to watch for changes. Directories are watched recursively. Can be given multiple times.
-    #[arg(long = "watch")]
+    /// A path to watch for changes.
+    ///
+    /// Directories are watched recursively. Can be given multiple times.
+    #[arg(long = "watch", value_name = "PATH")]
     pub paths: Vec<NormalPath>,
 
-    /// Reload the `ghci` session when paths matching this glob change. Can be given multiple
-    /// times. The last matching glob will determine if a reload is triggered.
+    /// Reload the `ghci` session when paths matching this glob change.
     ///
     /// By default, only changes to Haskell source files trigger reloads. If you'd like to exclude
     /// some files from that, you can add an ignore glob here, like `!src/my-special-dir/**/*.hs`.
     ///
     /// Globs provided here have precisely the same semantics as a single line in a `gitignore`
     /// file (`man gitignore`), where the meaning of `!` is inverted: namely, `!` at the beginning
     /// of a glob will ignore a file.
+    ///
+    /// The last matching glob will determine if a reload is triggered.
+    ///
+    /// Can be given multiple times.
     #[arg(long = "reload-glob")]
     pub reload_globs: Vec<String>,
 
-    /// Restart the `ghci` session when paths matching this glob change. Can be given multiple
-    /// times.
+    /// Restart the `ghci` session when paths matching this glob change.
     ///
     /// By default, only changes to `.cabal` or `.ghci` files or Haskell source files being
     /// moved/removed will trigger restarts.
     ///
-    /// Due to a `ghci` bug, the `ghci` session must be restarted when Haskell modules are removed
-    /// or renamed: https://gitlab.haskell.org/ghc/ghc/-/issues/11596
+    /// Due to [a `ghci` bug][1], the `ghci` session must be restarted when Haskell modules are removed
```

Comment:
Kinda clumsy to have markdown link syntax in the `--help` output like this but it looks OK enough and it renders correctly as Markdown.

It'd be nice to have a system that let us do some more advanced templating to add more detail in the manual...